packages/select/src/FilterMultiSelect/components/NoResults/NoResults.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from "react"
+import { Paragraph } from "@kaizen/typography"
+import styles from "./NoResults.module.scss"
+
+interface NoResultsProps {
```

Comment:
```suggestion
interface NoResultsProps extends HTMLAttributes<HTMLDivElement> {
```

Then spread the rest of the props into the container :)